sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -164,106 +164,153 @@ func _main() int {
 		return 0
 	}
 
-	// Clean up target container if it already exists before starting container
-	// task.
-	if err := deleteCtrIfExists(ctx, client, containerID); err != nil {
-		return 1
-	}
-
-	// Get the cgroup path of the systemd service
-	cgroupPath, err := cgroups.GetOwnCgroup("name=systemd")
+	// Check if the target container already exists. If it does, take over the helm to manage it.
+	container, err := client.LoadContainer(ctx, containerID)
 	if err != nil {
-		log.G(ctx).WithError(err).Error("Failed to discover systemd cgroup path")
-		return 1
+		if errdefs.IsNotFound(err) {
+			log.G(ctx).WithField("ctr-id", containerID).Info("Container does not exist, proceeding to create it")
```

Comment:
Nit, seems like you went to the trouble of unifying on lowercase elsewhere?

```suggestion
			log.G(ctx).WithField("ctr-id", containerID).Info("container does not exist, proceeding to create it")
```

Reply:
I was trying to follow the golang style guide regarding error strings https://github.com/golang/go/wiki/CodeReviewComments#error-strings. 

It also explicit says that this doesn't apply to normal log messages. So I kept info logs the same.